### ISSUE TITLE
"Invalid response from FloydHub server" on `data upload` attempt.

### ISSUE THREAD

**Issue by joeddav** [floydhub/floyd-cli#30]
When attempting to run `floyd data upload` the data compresses, uploads, and then hangs for a minute or so before outputting the invalid response error:

```
Creating data source. Total upload size: 48.8MiB
Uploading compressed data ...
Error: Invalid response from FloydHub server.1204312 - 00:00:00
```

Running on macOS Sierra: 10.12.5

**Comment by mckayward** [comment]
Hey @joeddav. This is a known issue that is being actively worked on, although it typically pops up with uploads larger than yours appears to be. The surest work around is to create a script that you can run inside of floyd to download and arrange your data, and then save it to `/output`. Something like: 
```
$ floyd run "python my_data_pulling_script.py"
```
Is that an option in your case?

**Comment by joeddav** [comment]
It's possible, just very inconvenient as it requires using a remote host for my data. But that's fine - I'll look forward to that issue being fixed and use that workaround in the meantime.

**Comment by mckayward** [comment]
Hey @joeddav,
This should be fixed with version 0.9.8 (released today). Is it working for you?
